### ISSUE TITLE
New Icon: Bloglovin

### ISSUE THREAD

**Issue by KokkieH** [Automattic/social-logos#60]
Requested in https://en.forums.wordpress.com/topic/social-icons-9?replies=2

**Comment by drw158** [comment]
Hey @KokkieH , this repo doesn't necessarily represent the logos that are available in the social icons widget. I don't think that widget directly uses these icons.

**Comment by KokkieH** [comment]
@drw158 fair enough, though on a previous occasion I was advised to post here: pKdGS-1Ap-p2

If the P2 is better for tracking these requests I'm happy to move it back there again.

**Comment by drw158** [comment]
Keeping it here is probably fine. We need to create an issue for somehow sync it with the widget.
